packages/grid/_modules_/grid/hooks/features/useColumnResize.tsx

```diff
@@ -185,14 +186,21 @@ export const useColumnResize = (columnsRef: React.RefObject<HTMLDivElement>, api
     }
 
     let newWidth =
-      initialOffset.current + finger.x - colElementRef.current!.getBoundingClientRect().left;
+      initialOffset.current! +
+      (finger as CursorCoordinates).x -
+      colElementRef.current!.getBoundingClientRect().left;
     newWidth = Math.max(MIN_COL_WIDTH, newWidth);
 
     updateWidth(newWidth);
   });
 
-  const handleTouchStart = useEventCallback((event: React.TouchEvent<HTMLElement>) => {
+  const handleTouchStart = useEventCallback((event) => {
     // If touch-action: none; is not supported we need to prevent the scroll manually.
```

Comment:
We could move the comment back to `if (!doesSupportTouchActionNone()) {`.

Reply:
Done